runtime-linux.md

```diff
@@ -0,0 +1,3 @@
+## File descriptors
+By default, only the `stdin`, `stdout` and `stderr` file descriptors are kept open for the application by the runtime.
```

Comment:
On Fri, Aug 28, 2015 at 08:21:14AM -0700, Mrunal Patel wrote:
> Yes, what those fds point to depends on the TTY setting, but the
> main point being that these are open for the application process.

Ah, good point.

the standard streams should maybe not be in backticks.  Alternatively,
these existing occurrences should be in backticks [1,2] ;).

[1]: https://github.com/opencontainers/specs/blob/6a5a4454655b4ffacbba11d50471817bc11ed36c/runtime.md#hooks
[2]: https://github.com/opencontainers/specs/blob/6a5a4454655b4ffacbba11d50471817bc11ed36c/runtime.md#pre-start